examples/xusb.c

```
@@ -1096,7 +1096,7 @@ int main(int argc, char** argv)
 	if (((uint8_t*)&endian_test)[0] == 0xBE) {
 		printf("Despite their natural superiority for end users, big endian\n"
 			"CPUs are not supported with this program, sorry.\n");
-		return 0;
+		return EXIT_SUCCESS;
```

Even though we returned "success" here it should return a failure in this case.